### ISSUE TITLE
Build: Add a dockerfile

### ISSUE THREAD

**Issue by carlaKC** [bitcoin-dev-project/sim-ln#89]

**Comment by maxwedwards** [comment]
An image on Docker Hub would also be very useful!

**Comment by Extheoisah** [comment]
Can I get clarity on what we want from this?

**Comment by carlaKC** [comment]
> Can I get clarity on what we want from this?

I started on a`Dockerfile` file here: https://github.com/carlaKC/sim-ln/tree/dockerfile

I think that all we need to add is instructions on how to mount a volume to provide the artifacts the the config file points to. Feel free to pick up from that branch as a starting point if you're interested!

**Comment by carlaKC** [comment]
@maxwedwards do you have any preferences/insights for what would be a convenient way to deal with macaroons/certs etc in the docker context (since the simulator can't just read them from local paths). 

Thinking something along the lines of: 
- Volume with all macaroons/certs in it. 
- Config should expect everything to be in `~/config`.

**Comment by maxwedwards** [comment]
I'm not sure anything specific needs to be added to the Dockerfile as when running the image couldn't you could mount a volume on any path and then specify that path in the config file?

**Comment by carlaKC** [comment]
> I'm not sure anything specific needs to be added to the Dockerfile as when running the image couldn't you could mount a volume on any path and then specify that path in the config file?

Yeah not to the image itself, just in our instructions for running with docker. Trying to make writing the config file as painless as possible, because I think paths/IPs generally trip folks up.

**Comment by maxwedwards** [comment]
I agree, it often trips me up!

I like your original proposal however not so sure on expecting things to be in a specific folder vs allowing the user to place anywhere. Perhaps just a solid example in the documentation?

**Comment by carlaKC** [comment]
Closed by #114.
